platform/broadcom/sonic-platform-modules-delta/ag9064/config-delta-ag9064.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
```

Comment:
this should going to sonic-linux-kernel repo.

Reply:
I will push it to kernel repo